siliconcompiler/core.py

```diff
@@ -2771,7 +2771,8 @@ def _find_leaves(self, steplist):
         return all_tasks.difference(non_leaf_tasks)
 
     ###########################################################################
-    def summary(self, steplist=None, show_all_indices=False):
+    def summary(self, steplist=None, show_all_indices=False,
+                generate_screenshot=True, generate_html=True, generate_pdf=False):
```

Comment:
Perhaps remove `generate_pdf` for now if it's currently non-functional, to prevent user confusion.

Reply:
Sure, can't hurt.